### ISSUE TITLE
the formula of the loss function and class prediction

### ISSUE THREAD

**Issue by QueenJuliaZxx** [qqwweee/keras-yolo3#44]
Hi,I had read the code,but I still cannot summaries the formula of the loss function of each part(bounding box,class_loss,object_loss),and beyond that I want to know the probability of the object and the class,thank you for your kindness!

**Comment by gyqiang** [comment]
Do you know the loss function now？
